nck/readers/google_dcm/config.py

```diff
@@ -48,3 +54,28 @@
     "YEAR_TO_DATE",
     "YESTERDAY",
 ]
+
+
+class GoogleDCMReaderConfig(BaseModel):
```

Comment:
Is it really a config file. There's a narrow gap between config and validation in this case. What do you think?

Reply:
It is a config file for me as it is specifying the structure of the class with types, ... But yes it's also doing validation so I am not sure about the best name/place for this type of classes. 
I think the first and most important feature of the class is still formatting all the json given. So to my mind, the class name should be good and the file too. What would be your advice on this?

Reply:
The name of the file itself has been defined during the reorganization of ACK. I just thought it was the better place for this function, especially because it's using the file's constants to build it's own config.

Reply:
Yes makes sense! We'll see in the future if we need to split the list of attributes from the validation methods 🙂